### PR TITLE
Fix a bug that `identify.rb` example doesn't work

### PR DESCRIPTION
**Pull request by Watson1978**
Fix https://github.com/rmagick/rmagick/issues/573

This PR will change following things.

1. Use valid constant. (GrayColorspace -> GRAYColorspace)

2. Use Kernel.sprintf instead of Kernel.format. 
Because this example adds method in `Magick::Image` using monkey patch.
However,  `Magick::Image` already `format` method as property and this example calls `Magick::Image#format`, after then raises ArugmentError exception.

### CODE REVIEW

**Comment by Watson1978** [comment]
Now, this example works well for me.

```
$ ruby identify.rb ../doc/ex/images/Flower_Hat.jpg
Image: ../doc/ex/images/Flower_Hat.jpg
	Format: JPEG
	Geometry: 200x250
	Class: DirectClass
	Type: TrueColorType
	Endianess: UndefinedEndian
	Colorspace: sRGBColorspace
	ChannelDepth:
	Channel statistics:
	Colors: 28253
	Rendering-intent: PerceptualIntent
	Gamma: 0.45454543828964233
	Resolution: 240x240
	Units: PixelsPerInchResolution
	Filesize: 10kb
	Interlace: NoInterlace
	Background Color: white
	Border Color: #DFDFDF
	Matte Color: gray74
	Page geometry: 200x250+0+0
	Dispose: UndefinedDispose
	Iterations: 0
	Compression: JPEGCompression
	Quality: 75
	Orientation: UndefinedOrientation
	comment: File written by Adobe Photoshop� 4.0
	date:create: 2018-09-07T13:30:59+09:00
	date:modify: 2018-09-07T13:30:59+09:00
	jpeg:colorspace: 2
	jpeg:sampling-factor: 2x2,1x1,1x1
	signature: dc7325b61c1316126ccf1c240933a01e3aff9ab8e20d693e2dec105fbb1a8a91
	Tainted: False
	Version: RMagick 3.1.0
	         ImageMagick 6.9.10-27 Q16 x86_64 2019-02-15 https://imagemagick.org
```

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -425,6 +425,10 @@ Style/EvalWithLocation:
     - 'lib/rmagick_internal.rb'
     - 'lib/rvg/units.rb'
 
+Style/FormatString:
```
Comment (dlemstra) [inline .rubocop_todo.yml]: Why was this excluded? Wouldn't it be easier to follow the style rules?

Comment (Watson1978) [reply .rubocop_todo.yml]: Please see the reason 2 in description. it need to fix the problem.

Comment (Watson1978) [reply .rubocop_todo.yml]: And I don't like this Rubocop's rules for formatting at least.
`format` is just alias of `sprintf` for me.

https://github.com/ruby/ruby/blob/e804fcb42c56d5a4ff31df0391accaf5ede8c26e/object.c#L4181-L4182

**Pull request merged**